group_vars/all

```diff
@@ -25,3 +25,10 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+# should mons also run osds
+mon_osd_colocation_allowed: "true"
+
+# OSDs could be run without netplugin on them
+# osd_contiv_network_include controls this behaviour
+osd_contiv_network_include: "true"
```

Comment:
well negation is kind of hard to read... also instead of mon and osd being capabilities I prefer more generic capabilities like compute and storage..mon and osd makes it specific to ceph and we will need something similar if we were to do NFS based storage on UCS B-series servers